quarkus/config-api/src/main/java/org/keycloak/config/ManagementOptions.java

```diff
@@ -61,7 +61,18 @@ public class ManagementOptions {
             .defaultValue("0.0.0.0")
             .build();
 
+    public enum Scheme {
+        http,
+        inherited
+    }
+
     //HTTPS
+    public static final Option<Scheme> HTTP_MANAGEMENT_SCHEME = new OptionBuilder<>("http-management-scheme", Scheme.class)
+            .category(OptionCategory.MANAGEMENT)
+            .description("Configures the management interface scheme. If 'inherited', the management interface will inherit the HTTPS settings of the main interface. If 'http', the management interface will be accessible via HTTP - it will not inherit HTTPS settings and cannot be configured for HTTPS.")
```

Comment:
```suggestion
            .description("Configures the management interface scheme. If 'inherited', the management interface will inherit the HTTPS settings of the main interface. If 'http', the management interface will be accessible only via HTTP - it will not inherit HTTPS settings and cannot be configured for HTTPS.")
```

@shawkins Yep, better, thanks!

nitpick that could clarify things a tiny bit more